Flaky test_parallel_replicas_distributed_read_from_all

https://s3.amazonaws.com/clickhouse-test-reports/0/e8d8c6dea58b1bff67148f5ff02738f01016c977/integration_tests__release__[3_4].html

```
>       assert (
            nodes[0].query(
                f"SELECT count(), min(key), max(key), sum(key) FROM {table_name}_d"
            )
            == expected_result
        )
E       AssertionError: assert '40\t0\t59\t1080\n' == '60\t0\t59\t1770\n'
E         - 60	0	59	1770
E         ? ^ 	 	  	 ^^
E         + 40	0	59	1080
E         ? ^ 	 	  	 ^^

```

--- a/tests/integration/test_parallel_replicas_distributed_read_from_all/test.py
+++ b/tests/integration/test_parallel_replicas_distributed_read_from_all/test.py
@@ -143,6 +143,8 @@ def test_read_equally_from_each_replica(start_cluster, prefer_localhost_replica)
     nodes[0].query(f"system start fetches {table_name}")
     nodes[1].query(f"system start fetches {table_name}")
     nodes[2].query(f"system start fetches {table_name}")
+    # ensure that replica in sync before querying it to get stable result
+    nodes[0].query(f"system sync  replica {table_name} strict")
     assert (
         nodes[0].query(
             f"SELECT count(), min(key), max(key), sum(key) FROM {table_name}_d"

--- a/tests/integration/test_parallel_replicas_distributed_read_from_all/test.py
+++ b/tests/integration/test_parallel_replicas_distributed_read_from_all/test.py
@@ -144,10 +144,14 @@ def test_read_equally_from_each_replica(start_cluster, prefer_localhost_replica)
     nodes[1].query(f"system start fetches {table_name}")
     nodes[2].query(f"system start fetches {table_name}")
     # ensure that replica in sync before querying it to get stable result
-    nodes[0].query(f"system sync  replica {table_name} strict")
+    nodes[0].query(f"system start merges {table_name}")
+    nodes[0].query(f"system sync  replica {table_name}")
     assert (
         nodes[0].query(
-            f"SELECT count(), min(key), max(key), sum(key) FROM {table_name}_d"
+            f"SELECT count(), min(key), max(key), sum(key) FROM {table_name}_d",
+            settings={
+                "allow_experimental_parallel_reading_from_replicas": 0,
+            }
         )
         == expected_result
     )

--- a/tests/integration/test_parallel_replicas_distributed_read_from_all/test.py
+++ b/tests/integration/test_parallel_replicas_distributed_read_from_all/test.py
@@ -151,7 +151,7 @@ def test_read_equally_from_each_replica(start_cluster, prefer_localhost_replica)
             f"SELECT count(), min(key), max(key), sum(key) FROM {table_name}_d",
             settings={
                 "allow_experimental_parallel_reading_from_replicas": 0,
-            }
+            },
         )
         == expected_result
     )

